internal/converter/internal/otelcolconvert/testdata/deltatocumulative.yaml

```diff
@@ -0,0 +1,19 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+processors:
+  deltatocumulative:
+
+exporters:
+  otlp:
+    endpoint: database:4317
+
+service:
+  pipelines:
+    metrics:
```

Comment:
The Yaml and Alloy pipelines are not equivalent. The Yaml one only has a metrics pipeline. The Alloy one has metrics, logs, and traces.